### PR TITLE
Add recently posted jobs to employer profile page

### PR DESCRIPTION
**Pull request by nvvu99**
## Related Tickets
- [#39829](https://edu-redmine.sun-asterisk.vn/issues/39829)

## WHAT (optional)
Add recently posted jobs to employer profile page

## Evidence (Screenshot or Video)
Employer profile

### CODE REVIEW

**Review by vuminhhieu1311 — APPROVED** [review]
Ok

**Review by vuminhhieu1311 — COMMENTED** [review]
Ô 2 commits à anh

**Comment by nvvu99** [comment]
> Ô 2 commits à anh

Có 1 commit của pull trước. Đợi merge vào thì chỉ còn 1 thôi.

**Comment by nvvu99** [comment]
ready

File: app/Models/EmployerProfile.php [hunk 1]
```diff
@@ -29,6 +29,13 @@ public function user()
...
     {
-        return $this->hasMany(Job::class)->as('applications');
+        return $this->hasMany(Job::class);
+    }
+
+    public function recentJobs()
+    {
+        return $this->jobs()
+                    ->orderBy('created_at', 'desc')
+                    ->limit(config('user.num_top_recent_jobs'));
```
Comment (huyenttm-0901) [inline app/Models/EmployerProfile.php:39]: indent 4 spaces

File: resources/views/employer/jobs.blade.php [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                                <div class="table-list-title">
+                                    <a href="{{ route('jobs.show', ['job' => $job]) }}">
+                                        <h3>{{ $job->title }}</h3>
+                                    </a>
+                                </div>
+                            </td>
+                            <td>
+                                <span class="applied-field">
+                                    <a href="{{ route('jobs.candidates', ['job' => $job]) }}">
+                                        {{ $job->employeeProfiles->count() }} {{ __('messages.applied') }}
```
Comment (huyenttm-0901) [inline resources/views/employer/jobs.blade.php:29]: câu hỏi như pull trc nhé. n+1 query

**Comment by nvvu99** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**